src/main/resources/org/jenkinsci/plugins/workflow/flow/GlobalDefaultFlowDurabilityLevel/help-durabilityHint.html

```diff
@@ -0,0 +1,20 @@
+<div>
+    <p>
+        This setting provides a trade-off between how fast pipelines run and their "durability", or ability to survive unexpected failures while running.
+
+        <br><strong>Facts:</strong>
+        <ul>
+            <li>The lowest durability modes (suggested) can be MUCH faster and write to disk FAR less often.</li>
+            <li>All pipelines persist their data when they complete/fail, are paused, or the master restarts gracefully.</li>
```

Comment:
There's some verbiage in the setting description strings and this which refer to a "clean shutdown" or "graceful restart" which are somewhat ambiguous terms in my book. Is sending the JVM a SIGTERM considered a "clean shutdown" for the purposes of Pipeline durability?

Assuming it is, I suggest defining that somewhere in this help document (or hyperlinked elsewhere).